### ISSUE TITLE
Free Action Potion not working against players

### ISSUE THREAD

**Issue by jeanjeanbon** [elysium-project/bug-tracker#506]
You can get Hamstring / Cone of cold while using a free action potion as a mage.

ex : screenshot before getting CoC https://i.imgur.com/Q3c6WFR.png

and after : https://i.imgur.com/OvL8Kc6.png

.

**Comment by jeanjeanbon** [comment]
Wow such enthusiasm.
I'm adding a video aswell 
https://www.youtube.com/watch?v=RVYAfcb6shs

**Comment by Navashed** [comment]
PTR was down for 2 weeks. Thanks for your patience.

Confirmed.

Free Action (Spell 6615) from Free Action Potion (Item 5634) is not currently providing immunity to slows from players. It is working correctly against slows from NPCs.

I was able to reproduce this solo by reflecting Frostbolt onto myself:
![freeaction](https://user-images.githubusercontent.com/2709545/37225032-4568cf22-23a3-11e8-9b93-a54446e9f72e.jpg)
